report/chap/formalism.tex

```diff
@@ -1,209 +0,0 @@
-\chapter{Games}
-We will look at a class of two player games on graphs. The two players will be called Player 0 and Player 1 (abbreviated as \Player{0}, \Player{1}). Although we will only look at specific games later, all of them fit into a general framework which will be discussed now.
-\section{Formalism}
-\label{sec:formalism}
-$G=(V_0,V_1,E)$ will denote a directed graph with vertex set $V=V_0 \sqcup V_1$ and edge set $E \subseteq V\times V$. $(V_0,V_1)$ is a partition of $V$ into vertices of P0 and P1 respectively. Always assume that $G$ has no dead ends ($\forall v \in V \exists w  (v,w) \in E$). $\PF$ will denote the set of all finite paths in $G$, and by $\PI$ the set of all infinite paths. $N(u)=\set{v \mid (u,v) \in E}$ is the set of outgoing neighbours of $u$.
```

Comment:
Don't start a sentence with a Math symbol. Sometimes doing so causes confusion - the full-stop can be confused for a \dot, for instance. Also, I think it becomes easier to read if the sentence starts with a word.